.github/workflows/action.yml

```diff
@@ -39,6 +41,8 @@ jobs:
     name: Expected local npm actions
     runs-on: ubuntu-latest
     if: github.ref != 'refs/heads/main'
+    permissions:
+      contents: read
```

Comment:
This was GitHub. If you prefer I can push in a separate pull request to separate concerns.